src/uct/ib/rc/base/rc_iface.h

```diff
@@ -283,6 +284,8 @@ struct uct_rc_iface {
         uint16_t             fc_wnd_size;
         uint8_t              fc_enabled;
 
+        uint32_t             conn_ece;
```

Comment:
check if moving to other place in config could reduce struct size

Reply:
Done.